### DISCUSSION TITLE
HTTP/1.1 403 Forbidden Error during route configuration tutorial

### DISCUSSION THREAD

**Discussion by igcompdocgmail** [apache/apisix#9682] [Q&A]
Hello, I am attempting to follow along with the Apache APISIX route configuration portion of the getting started tutorial. It seems in the most recent version of the tutorial 3.3, there isn't a conf.yaml file that I can troubleshoot with on my desktop. 

I am using a Git bash command line on Windows 11 with Docker Desktop and the container appears to be running properly.

Here is a snapshot of the curl command:
![curl-command](https://github.com/apache/apisix/assets/127332310/7e43e18a-370a-4a56-b507-3ba278e293fc)

Here is my command line error when trying to configure a route:
![Gitbas-error](https://github.com/apache/apisix/assets/127332310/f163e040-4dee-41ac-8111-bdc7593f28bd)

Relevant error logs from Docker Desktop:

![Docker-log](https://github.com/apache/apisix/assets/127332310/850f6744-4a8d-400e-a282-c432d6a5561d)

I also noticed that when trying to visit port 9180 on my browser I get a 404 not found:
![404](https://github.com/apache/apisix/assets/127332310/3b04be4c-9bbb-4f2d-a5cd-49f32ea51a4f)

Apologies, I am completely new to Docker and APISIX.

**Comment by igcompdocgmail** [comment]
I figured out the issue. I decided to just install apisix normally via the current installation guide, I then used the deprecated getting started tutorial and was able to create a route after changing the port in the old curl request to 9180.

**Reply by rizahmeds** [reply]
@igcompdocgmail Could be please share the link of **deprecated getting started tutorial** that you are talking above.
I am also facing the similar issue.

**Reply by igcompdocgmail** [reply]
@rizahmeds There is a version tab in the upper left corner on the Apache APISIX web page where you can select earlier versions to work with. I think if you follow the latest version it should likely work fine and I ended up not having to modify the default conf.yaml file at all. I was ultimately able to implement APISIX after having a little trouble getting acclimated to the software. The issue for me was that the mouse I have on my PC was using port 9080 for its software and that is actually what was causing the error. I would use your command line to double check that port 9080 is open and not already in use - I should have made a post clarifying this earlier.

**Reply by rizahmeds** [reply]
Thanks @igcompdocgmail for reverting back. Issue has been resolved. 

First i try checking `9080/9180 ports` have been occupied by some other process, but could not find any.
Then I deleted both existed docker containers(apisix and etcd) and did a fresh install using command:
 `curl -sL https://run.api7.ai/apisix/quickstart | sh`

Holla! Its working.
